src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaNotePersistenceAdapter.kt

```diff
@@ -19,38 +19,38 @@ class JpaNotePersistenceAdapter(
 
   @Transactional
   override fun createNote(createNoteDto: CreateNoteDto): NoteDto {
-    val noteEntity = noteRepository.save(NoteMapper.toEntity(createNoteDto))
-    return NoteMapper.toModel(noteEntity)
+    val noteEntity = noteRepository.save(NoteMapper.fromDomainToEntity(createNoteDto))
```

Comment:
I've renamed some of the methods in `NoteMapper` (below 👇 ) to make them more consistent with the other mappers in this package, and to make them clearer as to what they map from/to